airbyte-integrations/connectors/source-exact/setup.py

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from setuptools import find_packages, setup
+
+MAIN_REQUIREMENTS = [
+    "airbyte-cdk~=0.12",
+]
+
+TEST_REQUIREMENTS = [
+    "pytest~=6.1",
+    "pytest-mock~=3.6.1",
+    "source-acceptance-test",
+]
+
+DISCOVER_REQUIREMENTS = [
+    "beautifulsoup4~=4.11.1",
+    "html5lib~=1.1",
+    "pandas~=1.5.2",
+]
```

Comment:
Required for the `discover_schema.py` script to automatically write all json schemas